controllers/post.js

```diff
@@ -1,7 +1,14 @@
 const Post = require("../models/Post");
 
 exports.findAll = (req, res) => {
-  Post.find()
+  let filter = {};
+  if (req.query._id) {
+    filter = { _id: req.query._id };
+  } else {
+    let filter = {};
+  }
```

Comment:
else can be removed since filter is already set as an empty object

Reply:
Done